onyx-database-examples/lifecycle-events/src/main/java/com/onyxdevtools/lifecycle/AbstractDemo.java

```diff
@@ -1,97 +1,17 @@
 package com.onyxdevtools.lifecycle;
 
-import java.io.File;
-
-/**
- * Created by tosborn1 on 3/28/16.
- */
-public abstract class AbstractDemo {
-    /**
-     * Helper method to verify an the object is not null.
-     *
-     * @param message Message displayed if the object is null
-     * @param nonNullObject Object to assert
-     */
-    public static void assertNotNull(String message, Object nonNullObject)
-    {
-        if(nonNullObject == null)
-        {
-            System.err.println(message);
-        }
-    }
-
-    /**
-     * Helper method to verify an the object is null
-     *
-     * @param message Message displayed if the object is null
-     * @param nonNullObject Object to assert
-     */
-    public static void assertNull(String message, Object nonNullObject)
-    {
-        if(nonNullObject != null)
-        {
-            System.err.println(message);
-        }
-    }
-
-    /**
-     * Helper method to verify 2 objects have the same key
-     *
-     * @param message Message to display if the assert fails
-     * @param comparison1 First object to compare
-     * @param comparison2 Second object to compare against
-     */
-    public static void assertEquals(String message, Object comparison1, Object comparison2)
-    {
-        if(comparison1 == comparison2)
-        {
-            return;
-        }
-        if(comparison1.equals(comparison2))
-        {
-            return;
-        }
-
-        System.err.println(message);
-    }
+abstract class AbstractDemo {
 
     /**
      * Helper method to assert a boolean
      * @param message Message to show if assert fails
      * @param operator boolean to assert
      */
-    public static void assertTrue(String message, boolean operator)
+    static void assertTrue(String message, boolean operator)
     {
         if(!operator)
         {
             System.err.println(message);
```

Comment:
This is for an example project.  I do not want to obfuscate it with anything that does not have to do with Onyx